### PR TITLE
EWPP-2990: Drupal 10 compatibility.

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -35,28 +35,14 @@ services:
...
-    group: post-prepare
-    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
-    volumes:
-      - /cache:/cache
-    commands:
-      - composer update --prefer-lowest --prefer-stable --ansi --no-progress
-    when:
-      matrix:
-        COMPOSER_BOUNDARY: lowest
+      - composer require drupal/core:^${CORE_VERSION} drupal/core-composer-scaffold:^${CORE_VERSION} drupal/core-dev:^${CORE_VERSION} --update-with-all-dependencies --ansi --no-progress --no-interaction
```
Comment (drishu) [inline .drone.yml]:
```suggestion
      - composer require drupal/core:~${CORE_VERSION} drupal/core-composer-scaffold:~${CORE_VERSION} drupal/core-dev:^${CORE_VERSION} --update-with-all-dependencies --ansi --no-progress --no-interaction
```

File: .drone.yml [hunk 2]
```diff
@@ -77,9 +63,10 @@ pipeline:
...
 
 matrix:
-  COMPOSER_BOUNDARY:
-    - lowest
-    - highest
-  PHP_VERSION:
-    - 7.4
-    - 8.0
+  include:
+    - CORE_VERSION: 9.4
```
Comment (drishu) [inline .drone.yml]:
```suggestion
    - CORE_VERSION: 9.4.0
```

File: .drone.yml [hunk 3]
```diff
@@ -77,9 +63,10 @@ pipeline:
...
+      PHP_VERSION: 8.0
+    - CORE_VERSION: 9.4
```
Comment (drishu) [inline .drone.yml]:
```suggestion
    - CORE_VERSION: 9.4.0
```

**Review by drishu — CHANGES_REQUESTED** [review]
The rest of it looks good

**Review by drishu — APPROVED** [review]

File: .drone.yml (hunk 2 above)
Comment (22Alexandra) [reply .drone.yml]: Why tho? Patch versions usually include security updates and maybe we should test using the latest patch instead of `.0`.

Comment (22Alexandra) [reply .drone.yml]: Nvm.

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**